examples/asr/conf/conformer/conformer_stateless_transducer_bpe.yaml

```diff
@@ -0,0 +1,259 @@
+# It contains the default values for training a Conformer-Transducer ASR model with stateless decoder, large size (~120M) with Transducer loss and sub-word encoding.
```

Comment:
This file should be part of a subfolder - not directly inside of conformed. So conformer/stateless/...yaml